src/odemis/acq/calibration.py

```diff
@@ -179,47 +237,101 @@ def get_spectrum_efficiency(das):
     return ret
 
 
-def compensate_spectrum_efficiency(data, bckg=None, coef=None):
+def spectrum_efficiency_and_bg_correction(data, bckg=None, coef=None):
```

Comment:
It's not really a great name. Firstly, functions should start with a verb, and secondly it's a little too long. However, I don't have great idea either... Maybe just `adjust_spectrum` or `compensate_spectrum`? I don't think the name absolutely needs to indicate that it uses the efficiency coefficient and the background, because these are in the arguments of the function.

Reply:
How about `apply_corrections`?